src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -160,14 +159,17 @@ public Task RunAsync()
                     ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
                 }
 
+                ColorConsole.WriteLine();
+
+                var totalColor = currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0];
+                ColorConsole.Write(padding, $"{currentLeaderboard?.Count ?? 0:N0}".Color(totalColor));
+
                 if (currentLeaderboard.Since.HasValue)
                 {
                     ColorConsole.WriteLine(
-                        padding,
-                        $"{currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());
+                        $" from {currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());

```

Comment:
There are really easy to gloss over. Given when we run these things, is it fair to say that:
- the year is unlikely to be relevant
- the month is probably also irrelevant
- the last activity date/time is likely to be nowish
- the seconds are likely to be irrelevant (especially given whatever latency is inherent to the system)

Would it make sense to have the message say something like:

> 642 from Tuesday 13:05 to now